CONTRIBUTING.md

```diff
@@ -5,6 +5,25 @@
 
 _See also: [Flutter's code of conduct](https://github.com/flutter/flutter/blob/master/CODE_OF_CONDUCT.md)_
 
+## Important note
+
+As of January 2021, we are no longer accepting non-critical PRs for plugins
+for which there is a corresponding [Flutter Community Plus
+Plugin](https://plus.fluttercommunity.dev/), as we hope that in time we're able
```

Comment:
"as we hope in time to be able"

Reply:
Sure, done. Also made the same change in the snippet we're using to close PRs, since this was copied directly from there for consistency.

I'll also be sure to pass along your feedback to the original author of that snippet ;)

Reply:
Excellent. I'm sure they'll appreciate the feedback. : )